kernelci.org/content/en/docs/admin/api.md

```diff
@@ -7,39 +7,36 @@ description: "KernelCI API"
 ## Getting an admin token
 
 Some operations such as creating user accounts requires an admin user, so you
-need to be in the `admin` group.  Then you need to get an API token with the
-`admin` scope to perform queries restricted to admins.  Here are some sample
-commands to do that:
+need to be in the `admin` group.  Then you need to get an API token to perform
+queries restricted to admins.  Here are some sample commands to do that:
 
 ```
-$ kci user whoami
+$ kci user whoami --indent=1
 {
-    "id": "64f5ff9e8326c545a780c2a0",
-    "active": true,
-    "profile": {
-        "username": "<your-username>",
-        "hashed_password": "<your-hashed-password>",
-        "groups": [
-            {
-                "id": "6499aa9da02fef8143c1feb0",
-                "name": "admin"
-            }
-        ],
-        "email": "<your-email-address>"
+  "id": "64f5ff9e8326c545a780c2a0",
+  "email": "<your-email-address>",
+  "is_active": true,
+  "is_superuser": true,
+  "is_verified": true,
+  "username": "<your-username>",
+  "groups": [
+    {
+      "id": "6499aa9da02fef8143c1feb0",
+      "name": "admin"
     }
+  ]
 }
-$ kci user get_token --username=<your-username> --scopes=admin
+```
+
```

Comment:
It's missing some \`\`\` here or the line above shouldn't have one.  See the glitch on the rendered page:
https://static.staging.kernelci.org/docs/admin/api/

Reply:
Thanks for pointing it out. Fixed it.